app/jinja_filters.py

```diff
@@ -52,6 +52,29 @@ def format_percentage(value):
     return '{}%'.format(value)
 
 
+@blueprint.app_template_filter()
+def format_address_list(user_entered_address=None, metadata_address=None):
+    """
+        This function format_address_list accepts two addresses, a user submitted address and a metadata address
+        If all the items in address list are 'Undefined' (nothing in answer_store) then we retrieve the metadata.
+
+        :param user_entered_address user entered address
```

Comment:
More pickiness sorry, but could you expand these a bit more and say that they're lists of address fields.  Makes it easier to figure out what's going on at a glance